github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+
+-   All hierarchy is whack separated and no caps.
+>  
+
+| ***State/Status*** | ***Area***               | ***Issue Type*** | ***Pri***       |
+|--------------------|--------------------------|------------------|-----------------|
+| investigating      | area-common              | question         | Pri-0,(regular) |
+| needs-more-info    | areas-communication      | enhancement      |                 |      
+| understood         | area-data-structures     | code-defect      |                 |
+| in-design          | area-developer           | test-defect      |                 |
+| in-progress        | area-diagnostics         | doc-only         |                 |
+| external           | area-documentation       |                  |                 | 
+| release-note-na    | area-engineering         |                  |                 |
+| release-note-req   | area-federation          |                  |                 |
+| duplicate          | area-hosting             |                  |                 |
+|                    | area-ktl                 |                  |                 |
+|                    | area-linux               |                  |                 |
+|                    | area-management          |                  |                 |
+|                    | area-operations          |                  |                 |
+|                    | area-reliability         |                  |                 |
+|                    | area-security            |                  |                 |
+|                    | area-setup               |                  |                 |
+|                    | area-sfrp                |                  |                 |
+|                    | area-standaloneinstaller |                  |                 |
+|                    | area-store               |                  |                 |
+|                    | area-test                |                  |                 |
+|                    | area-testability         |                  |                 |
+|                    | area-transport           |                  |                 |
+|                    | area-needs-triage        |                  |                 |
+|                    | area-try-service-fabric  |                  |                 |
+|                    | varea-windows-server     |                  |                 |
+|                    |                          |                  |                 |
+
+Project - NA for now
+--------------------------------------------------------------------------
+
+| ***xxxx*** | ***xxx*** |
+|------------|-----------|
+| xxx        | xxx       |
+
+
+> **No labels for this area at this time.**
+
+Milestones
+----------------------------------------------------------------
+
+| ***Release (Iteration path)***                                                            |
```

Comment:
it might help to have some background information for laymen on what you mean by "Iteration path" and "CU1".  I don't really have a guide idea of how releases work, and external people won't either.